doc/radosgw/lua-scripting.rst

```diff
@@ -6,10 +6,18 @@ Lua Scripting
 
 .. contents::
 
-This feature allows users to assign execution context to Lua scripts. The three supported contexts are ``preRequest``" which will execute a script before each
-operation is performed, ``postRequest`` which will execute after each operation is performed, and ``background`` which will execute within a specified time interval.
-A request context script may be constrained to operations belonging to a specific tenant's users.
-The request context script can also access fields in the request and modify some fields. All Lua language features can be used.
+This feature allows users to assign execution context to Lua scripts. The supported contexts are:
+
+ - ``preRequest`` which will execute a script before each operation is performed
+ - ``postRequest`` which will execute after each operation is performed
+ - ``background`` which will execute within a specified time interval
+ - ``getData`` which will execute on objects' data when objects are downloaded
```

Comment:
Warms my heart to see proper plural possessive

Reply:
thanks!